modules/gdnative/gdnative_api.json

```diff
@@ -6452,6 +6452,14 @@
 		  ["godot_object *", "p_obj"],
 		  ["const godot_net_multiplayer_peer *", "p_interface"]
           ]
+        },
+        {
+          "name": "godot_net_bind_webrtc_peer",
```

Comment:
This technically does not break binary compatibility, but the API struct for this extension will be "valid" even with Godot versions where this function does not exist.

So this should be in the "next" extension of the "NET" extension.